.circleci/real_config.yml

```diff
@@ -2650,6 +2666,14 @@ workflows:
               only:
                 - main
 
+      - check-docs-links:
+          requires:
+            - build-docs
+          filters:
+            branches:
+              only:
+                - main
```

Comment:
for clarification, we're setting this up only to apply on main.

I guess this is because we assume breakages are usually going to be external, and people won't be adding broken links in newly written docs?

Reply:
Right. I can try to figure out how to add a "do this if docs are changed too" filter, but I'm not convinced that it's necessary right now.

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths
```yaml
on:
  push:
    paths: docs/**
```

Reply:
Probably https://circleci.com/developer/orbs/orb/circleci/path-filtering

Stick that in the config.yaml and set another parameter for "docs change" or something like that, then use that parameter as a conditional for jobs in real_config.yaml

Reply:
For clarification, this is run based on `build-docs`, not `publish-docs`. This means it will be run on every `main` commit, and not just before release.